tests/gamma/csm_observability_test.py

```diff
@@ -29,6 +31,51 @@
 _XdsTestClient = xds_k8s_testcase.XdsTestClient
 _Lang = skips.Lang
 
+# Testing consts
+_TEST_RUN_SECS = 90
```

Comment:
let's drop `_` in all these consts. It's more distracting than helpful.

Reply:
Done